mkdocs.yml

```diff
@@ -1,5 +1,5 @@
 site_name: Percona Monitoring and Management
-site_url: https://www.percona.com/doc/percona-monitoring-and-management/2.x/
+#site_url: https://new.percona.com/doc/percona-monitoring-and-management/2.x/
```

Comment:
Would it work if `site_url` is commented out?

Reply:
Yes it does and means it works on both new.percona.com and www.percona.com